src/main/java/ma/boot/springboot/service/mapper/UserMapper.java

```diff
@@ -15,4 +16,8 @@ public User mapReviewDtoToUser(ReviewDto dto) {
         user.setPassword(DEFAULT_PASSWORD);
         return user;
     }
+
+    public UserResponseDto mapUserToDto(User user) {
+        return new UserResponseDto(user.getProfileName());
+    }
```

Comment:
do you use this?

Reply:
No, forgot to delete it - in previous solution (which didn't work out) I need it...